code/modules/mob/living/carbon/human/species/station/human_subspecies.dm

```diff
@@ -1,63 +1,43 @@
-
-	flash_mod =     0.9
-	oxy_mod =       1.1
-	radiation_mod = 0.5
-	brute_mod =     0.85
-	slowdown =      1
-	
-	spawn_flags = IS_RESTRICTED
+/datum/species/human/offworlder
+	name = "Off-Worlder Human"
+	name_plural = "off-worlders Humans"
+	blurb = "The Offworlders are humans that have adapted to zero-G conditions through a lifetime of conditioning, exposure, and physical modification. \
+	They thrive in thinner atmosphere and weightlessness, more often than not utilizing advanced life support and body-bracing equipment to sustain themselves in normal Human environments."
+	icobase = 'icons/mob/human_races/subspecies/r_offworlder.dmi'
 
-/datum/species/human/spacer
-	name = "Space-Adapted Human"
-	name_plural = "space-adapted Humans"
-	blurb = "Lithe and frail, these sickly folk were engineered for work in environments that \
-	lack both light and atmosphere. As such, they're quite resistant to asphyxiation as well as \
-	toxins, but they suffer from weakened bone structure and a marked vulnerability to bright lights."
-	icobase = 'icons/mob/human_races/subspecies/r_spacer.dmi'
-
-	oxy_mod =   0.8
-	toxins_mod =   0.9
-	flash_mod = 1.2
-	brute_mod = 1.1
-	burn_mod =  1.1
-	
-	spawn_flags = IS_RESTRICTED
+	flash_mod =     1.2
+	oxy_mod =       0.8
+	brute_mod =     1.2
+	toxins_mod =    1.2
+	bleed_mod = 0.5
+
+	warning_low_pressure = 30
+	hazard_low_pressure = 10
 
-/datum/species/human/vatgrown
-	name = "Vat-Grown Human"
-	name_plural = "vat-grown Humans"
-	blurb = "With cloning on the forefront of human scientific advancement, cheap mass production \
-	of bodies is a very real and rather ethically grey industry. Vat-grown humans tend to be paler than \
-	baseline, with no appendix and fewer inherited genetic disabilities, but a weakened metabolism."
-	icobase = 'icons/mob/human_races/subspecies/r_vatgrown.dmi'
-
-	toxins_mod =   1.1
-	has_organ = list(
-		"heart" =    /obj/item/organ/heart,
-		"lungs" =    /obj/item/organ/lungs,
-		"liver" =    /obj/item/organ/liver,
-		"kidneys" =  /obj/item/organ/kidneys,
-		"brain" =    /obj/item/organ/brain,
-		"eyes" =     /obj/item/organ/eyes
-		)
-		
 	spawn_flags = IS_RESTRICTED
 
-/*
-// These guys are going to need full resprites of all the suits/etc so I'm going to
-// define them and commit the sprites, but leave the clothing for another day.
-/datum/species/human/chimpanzee
-	name = "Uplifted Chimpanzee"
-	name_plural = "uplifted Chimpanzees"
-	blurb = "Ook ook."
-	icobase = 'icons/mob/human_races/subspecies/r_upliftedchimp.dmi'
-*/
+/datum/species/human/offworlder/equip_later_gear(var/mob/living/carbon/human/H)
+	if(H.back)
+		var/obj/item/I = H.back
+		H.unEquip(I)
+		H.put_in_hands(I)
+
+	var/obj/item/weapon/rig/light/offworlder/skeleton = new(get_turf(H))
+	skeleton.autodrobe_no_remove = TRUE
+	H.equip_to_slot_or_del(skeleton,slot_back)
+	to_chat(H, "<span class='notice'>You have access to \the [skeleton], deploy it to allow you to stand properly.</span>")
```

Comment:
This should probably auto-deploy so that they don't spawn flat on their faces.

Reply:
I did try, for whatever reasons, it does not deploy, but having the rig on your back slightly increases your chances of not falling down.